components/webapp-mgt/org.wso2.carbon.application.mgt.webapp.ui/src/main/resources/web/war-apps/jaxws_info.jsp

```diff
@@ -80,7 +80,7 @@
             <% if ("Faulty".equals(data.getState())) { %>
             <td><%= data.getContext()%></td>
             <% } else { %>
-            <td><a href="../webapp-list/webapp_info.jsp?webappFileName=<%= data.getWebappFileName()%>&webappState=<%= state%>&hostName=<%=data.getHostName()%>&httpPort=<%=data.getHttpPort()%>&webappType=jaxWebapp"><%= data.getContext()%></a></td>
+            <td><a href="../webapp-list/webapp_info.jsp?webappFileName=<%= data.getWebappFileName()%>&webappState=<%= state%>&hostName=<%=data.getHostName()%>&httpPort=<%=data.getHttpPort()%>&webappType=jaxWebapp&defaultHostName=<%=data.getHostName()%>"><%= data.getContext()%></a></td>
```

Comment:
This code line is a bit lengthy, better if we can break it into few lines. 
Fix code formatting.